otheatre/src/main/java/com/orange/otheatre/otheatre/exceptions/PlayEligibilityExceptions/EndTimeAfterException.java

```diff
@@ -0,0 +1,9 @@
+package com.orange.otheatre.otheatre.exceptions.PlayEligibilityExceptions;
+
+public class EndTimeAfterException extends Exception {
+
+    public EndTimeAfterException() {
+        super("The play can't end after 11:00PM!");
+    }
+
```

Comment:
We don't need in this context a particular exception. A Conflict exception is enough.